.github/workflows/pitest-update-pr.yml

```diff
@@ -0,0 +1,35 @@
+# Updates a pull request based on the corresponding mutation testing analysis
+# performed by the `pitest-analyze-pr.yml` workflow. See
+# https://blog.pitest.org/oss-pitest-pr/ for details.
```

Comment:
The changes in this file will only take effect once merged to the default branch. See the code reverted in the most recent commit to see how I tested a variation of this code.